webapp/packages/plugin-settings-panel/src/SettingsPanel/useSettings.ts

```diff
@@ -27,7 +28,13 @@ export function useSettings(accessor?: string[]): ISettings {
         const map = new Map();
         const settings = this.settingsManagerService.activeSettings
           .filter(setting => accessor?.some(value => setting.access.scope.includes(value)))
-          .sort((a, b) => a.name.localeCompare(b.name));
+          .sort((a, b) => {
+            if (a.type === b.type) {
+              return translate(a.name).localeCompare(translate(b.name));
+            } else {
+              return a.type - b.type;
+            }
+          });
```

Comment:
I'm really not sure that we need to group settings by type because it can look better in a small amount of settings but will be unreadable with many

Reply:
<img width="461" alt="image" src="https://github.com/user-attachments/assets/899006c1-dd73-4c81-acec-081bafeb8b93">
<img width="511" alt="image" src="https://github.com/user-attachments/assets/ef3a1fff-18d9-47eb-8232-d0470dad75ba">

For me settings of the same type is more likely to be closer by meaning. See how it looks w/o sorting by type